package.json

```diff
@@ -21,6 +21,7 @@
     "html-minifier": "^4.0.0",
     "http-link-header": "^1.0.2",
     "shimport": "^1.0.1",
+    "source-map": "^0.7.3",
```

Comment:
we don't need this anymore since you're including the file directly, right?

Reply:
We do need that one. It's used to do the actual mapping.